### PR TITLE
added terraform version in the command

### PR DESCRIPTION
**Pull request by yogesh266**
added terraform version in the command

### CODE REVIEW

File: ads/feature_store/common/spark_session_singleton.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def __init__(self, metastore_id: str = None):
         )
 
         if not developer_enabled() and metastore_id:
+            print("Not Developer Enabled")
```
Comment (KshitizLohia) [inline ads/feature_store/common/spark_session_singleton.py:65]: remove print statements

File: ads/feature_store/common/spark_session_singleton.py [hunk 2]
```diff
@@ -79,6 +80,7 @@ def __init__(self, metastore_id: str = None):
                 .config("spark.driver.memory", "16G")
 
         if developer_enabled():
+            print("Developer Enabled")
```
Comment (KshitizLohia) [inline ads/feature_store/common/spark_session_singleton.py:83]: remove debug statements

File: ads/feature_store/common/utils/utility.py [hunk 3]
```diff
@@ -204,7 +205,7 @@ def get_input_features_from_df(
 
 
 def convert_expectation_suite_to_expectation(
-    expectation_suite: ExpectationSuite, expectation_type: ExpectationType
+        expectation_suite: ExpectationSuite, expectation_type: ExpectationType
```
Comment (KshitizLohia) [inline ads/feature_store/common/utils/utility.py:208]: I'm unsure if this has been formatted using black. Could you kindly double-check

File: ads/feature_store/common/spark_session_singleton.py (hunk 2 above)
Comment (yogesh266) [reply ads/feature_store/common/spark_session_singleton.py]: done

File: ads/feature_store/common/utils/utility.py (hunk 3 above)
Comment (yogesh266) [reply ads/feature_store/common/utils/utility.py]: done

File: ads/feature_store/common/spark_session_singleton.py (hunk 1 above)
Comment (yogesh266) [reply ads/feature_store/common/spark_session_singleton.py]: done

**Review by KshitizLohia — APPROVED** [review]
LGTM

**Pull request merged**